mmdet/models/plugins/dropblock.py

```diff
@@ -79,3 +79,8 @@ def _compute_gamma(self, feat_size):
         factor = (1.0 if self.iter_cnt > self.warmup_iters else self.iter_cnt /
                   self.warmup_iters)
         return gamma * factor
+
+    def extra_repr(self):
+        s = ('drop_prob={drop_prob}, block_size={block_size}, '
```

Comment:
drop_prob={self.drop_prob}  ...

Reply:
@hhaAndroid I initially went for the same coding style than `extra_repr` of Conv in pytorch repo ([code here](https://github.com/pytorch/pytorch/blob/7e7be526c9d9179f35084e9cca5b5c5ad5172100/torch/nn/modules/conv.py#L150)). 
It is now replaced with your requested change.